src/ui/public/chrome/api/__tests__/xsrf.js

```diff
@@ -114,7 +113,7 @@ describe('chrome xsrf apis', function () {
       });
 
       it('accepts alternate tokens to use', function () {
```

Comment:
Ah, I know why: this test was never really testing anything useful.  It's just verifying that `$http` sends custom headers, which the actual tests for `$http` should verify for us.  I think you can safely remove this whole test.